src/rlbot.rs

```diff
@@ -87,6 +92,29 @@ impl RLBot {
         core_result(status)
     }
 
+    /// Sends player input to RLBot using flatbuffers.
+    pub fn update_player_input_flatbuffer(
```

Comment:
Nit: Since this is the low-level interface, I think the param order should match the original field order:

```
table PlayerInput {
  playerIndex:int;
  controllerState:ControllerState;
}
```

(not sure how I missed this the first time through)

Reply:
Done.